state-chain/cf-integration-tests/src/mock_runtime.rs

```diff
@@ -366,6 +366,7 @@ impl ExtBuilder {
 							usdc_token_mint_pubkey: sol_test_values::USDC_TOKEN_MINT_PUB_KEY,
 						},
 					),
+					block_shared_data_liveness: Default::default(),
```

Comment:
(and everywhere else "liveness" is used here)
```suggestion
					block_shared_data_lifetime: Default::default(),
```